packages/components/src/components/dialog/dialog.scss

```diff
@@ -43,9 +60,9 @@
         justify-content: space-around;
         flex-direction: column;
         align-items: flex-start;
+        box-shadow: 0 2px 8px 0 $COLOR_LIGHT_BLACK_3;
```

Comment:
I think the shadow color in dark & light theme is different. isn't it?

Reply:
@reza-fs it was like this before so i assume its how it supposed to be.

Reply:
@Mahboobeh-binary agree with @reza-fs ,  the box-shadow color is different in dark & light theme